check_imagestreams.py

```diff
@@ -0,0 +1,63 @@
+#! /bin/python
```

Comment:
Having my Python hat on, shebangs should always have version. Another thing is that on some systems (RHEL-7), the python might be provided by SCL, so the shebang might be `#!/bin/env python3`. For example on our RHEL-7 CI machine, the python3 is available through virtualenv:
```
# which python3
/usr/local/python36-tools/bin/python3
```

Reply:
And my stupid typo in shebang;)